docs/source/features/mixed_precision.rst

```diff
@@ -27,25 +68,46 @@ NVIDIA H100 GPU introduced support for a new datatype, FP8 (8-bit floating point
 
    * - Argument
      - Description
-   * - transformer_engine
-     - TE and related functionality can be enabled by setting this boolean argument to True. If this argument is not set to True, all subsequent arguments will be ignored.
    * - fp8
-     - Enables FP8 training. For transformer networks, the QKV, projection, FC1, and FC2 matrix multiplications are executed using the fourth-generation NVIDIA H100 Tensor Cores with FP8 support.
-   * - fp8_e4m3
-     - Training recipe format for FP8. Activations, weights, and gradient tensors use the E4M3 format.
-   * - fp8_hybrid
-     - Training recipe format for FP8. Activations and weight tensors use the E4M3 format, whereas gradient use the E5M2 format to satisfy the additional dynamic range requirement for backward tensors. This is the default setting.
+     - Training recipe format for FP8: 'hybrid' or 'e4m3'. 'hybrid' is set by default. With 'hybrid', activations and weight tensors use the E4M3 format, whereas gradient use the E5M2 format to satisfy the additional dynamic range requirement for backward tensors.
    * - fp8_margin
      - The scaling factor for FP8 tensors can be shifted by a factor of $2 ^ {margin}$ using this argument.
    * - fp8_amax_history_len
      - Window size for amax history. The window size determines how many instances of the most recent absolute max values (amaxes) are stored per tensor.
    * - fp8_amax_compute_algo
      - The choice between “max” and “most_recent” specifies how to select an amax value from the given history.
-   * - reduce_amax
-     - Indicates whether or not to perform an allreduce on the amax (absolute max) values for the FP8 tensors. Since the amax is directly used to compute the scaling factor for FP8 tensors, setting this argument ensures that the scaling factors for a tensor remain synchronized across devices in multi-GPU training configurations.
    * - fp8_params
      - Indicates whether to store module-level parameters in FP8. Enabling this option can reduce memory consumption by eliminating the need to store a copy of weights in higher precision for cases where these weights are externally maintained, such as master parameters in the optimizer. For more information, refer to the `fp8_model_init <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/pytorch.html#transformer_engine.pytorch.fp8_model_init>`_ API in TE.
 
+Usage
```

Comment:
Replace the "Usage" heading with "Implement FP8 Training" or "Apply FP8 Training".

Implement FP8 Training
^^^^^^^^^^^^^^^